src/Azure.DataApiBuilder.Core/Configurations/RuntimeConfigValidator.cs

```diff
@@ -767,6 +764,7 @@ public void ValidateRelationshipsInConfig(RuntimeConfig runtimeConfig, ISqlMetad
         /// Validates the parameters given in the config are consistent with the DB i.e., config has all
         /// the parameters that are specified for the stored procedure in DB.
         /// </summary>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "Current use by dab workflow")]
```

Comment:
Is there a need to add this in the current PR?

Reply:
Same as below